novawallet/Modules/NetworkManagement/CustomNetwork/Interactor/CustomNetworkBaseInteractor.swift

```diff
@@ -235,63 +280,66 @@ private extension CustomNetworkBaseInteractor {
             case let .success(chain):
                 self?.handleSetupFinished(for: chain)
             case let .failure(error as CustomNetworkSetupError) where error == .decimalsNotFound:
-                self?.basePresenter?.didReceive(.common(innerError: .noDataRetrieved))
+                self?.presenter?.didReceive(.common(innerError: .noDataRetrieved))
             default:
-                self?.basePresenter?.didReceive(.common(innerError: .undefined))
+                self?.presenter?.didReceive(.common(innerError: .undefined))
             }
         }
     }
-    
+
     // MARK: Optional helpers
-    
+
     func extractPriceId(from coingeckoUrlTemplate: String?) -> AssetModel.PriceId? {
         guard let coingeckoUrlTemplate else { return nil }
-        
+
         let regex = try? NSRegularExpression(pattern: Constants.priceIdSearchRegexPattern)
         let range = NSRange(location: 0, length: coingeckoUrlTemplate.utf16.count)
-        
+
         guard
             let match = regex?.firstMatch(in: coingeckoUrlTemplate, options: [], range: range),
             let matchedRange = Range(match.range(at: 1), in: coingeckoUrlTemplate)
         else {
             return nil
         }
-                
+
         return String(coingeckoUrlTemplate[matchedRange])
     }
-    
+
     func createExplorer(from url: String?) -> ChainModel.Explorer? {
-        guard let url = url?.trimmingCharacters(in: CharacterSet(charactersIn: "/")) else {
+        guard let url = url?.trimmingCharacters(
+            in: CharacterSet(charactersIn: "/").union(CharacterSet.whitespaces)
```

Comment:
`.slash` is a case of the `String.Separator` enum so it might be a bit misleading to use `String.Separator.slash.rawValue`. Also, we will have to wrap it with `String`, cause `.slash` has `Character` type and `CharacterSet` init needs `String` value. I find the result being too verbose in that case